gunicorn/_compat.py

```diff
@@ -50,7 +50,8 @@ def _get_codeobj(pyfile):
         # So to get code object, just read the 8th byte onwards till EOF, and
         # UN-marshal it.
         import marshal
-        code_obj = marshal.loads(data[8:])
+        header_size = 12 if PY33 else 8
```

Comment:
Could you add a `# TODO` comment so we don't forget to remove this check when we start to develop 20.0?